src/Cache/TwigTemplateRecorderInjector.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CacheBundle\Cache;
+
+use Sonata\Cache\Invalidation\Recorder;
+use Twig_Environment;
+use Twig_Extension;
+use Twig_Extension_InitRuntimeInterface;
+
+/**
+ * @internal
+ *
+ * NEXT_MAJOR: remove
+ */
+final class TwigTemplateRecorderInjector extends Twig_Extension implements Twig_Extension_InitRuntimeInterface
```

Comment:
please add `@author` tag

Reply:
Who do I put as author? Myself? Or someone else (since I didn't write the original code)

Reply:
Oh i thought you did, in this case you ca ignore my comment

Reply:
No, I just moved the code from `Bundle::boot()` to when twig will actually initialize